spec/spec_helper.rb

```diff
@@ -1,3 +1,4 @@
+require 'set'
```

Comment:
I guess you never tried running `bundle exec rspec` locally because it didn't work 👀 hahahahaha

Reply:
Runs fine for me:

```
➜  boring-service git:(main) bundle exec rspec
............

Finished in 0.00403 seconds (files took 0.05898 seconds to load)
12 examples, 0 failures
```

Reply:
maybe you're importing the default gems somehow on your end? some special setup/customization you have on your machine? IDK

this is how it looks a fresh `git clone` + `bundle` + `bundle exec rspec`
![Uploading image.png…]()